diff --git a/gizmo/text/utf8.h b/gizmo/text/utf8.h
index 7c3bcdb..99f1a92 100644
--- a/gizmo/text/utf8.h
+++ b/gizmo/text/utf8.h
@@ -2,6 +2,7 @@
 #define __UTF8_H__

 #include <string>
+#include <cstdint>


 class Utf8

diff --git a/gizmo/text/ssa.h b/gizmo/text/ssa.h
index 66d9918..170e265 100644
--- a/gizmo/text/ssa.h
+++ b/gizmo/text/ssa.h
@@ -5,6 +5,7 @@
 #include <list>
 #include <set>
 #include <string>
+#include <cstdint>


 class SSAParser
